examples/archive.html

```diff
@@ -155,6 +163,10 @@ <h3 id="offcanvasRightLabel">Images</h3>
         });
     });
 
+    saveButton.addEventListener('click', function() {
+      alert(placedImagesCollection.generateExportJson()); // we'll do something better than alert in the next PR
```

Comment:
OK, so this worked! I tested it in GitPod. But it outputs "[object, object]" so we need to stringify the JSON object:

```suggestion
      alert(JSON.stringify(placedImagesCollection.generateExportJson())); // we'll do something better than alert in the next PR
```

Reply:
Hii @jywarren 
I’ve been trying to test it on gitpod 
But I don’t know how to view the live of just the archive.html 
It shows me the map of the index.html using gitpod 
Please how did you test it with gitpod?

Reply:
OK, so what I found was that, for some reason, this doesn't work... the images array is empty. But! I also found that we actually already have an distortableImageCollection in this page -- on line 96!

So on this line, we can do:

```js
      alert(JSON.stringify(map.imgGroup.generateExportJson())); // we'll do something better than alert in the next PR
```

And we can also remove the lines on 76 and 77, since we are just going to use the Collection that's already defined in the page. What do you think?